DigitalOffice/ProjectService/Startup.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using ProjectService.Database;
+
+namespace ProjectService
+{
+    public class Startup
+    {   
+        public IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<ProjectServiceDbContext>(options =>
+            {
+                options.UseSqlServer(Configuration.GetConnectionString("SQLConnectionString"));
+            });
+
+            services.AddControllers();
+        }
+        
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
+        }
```

Comment:
How about `app.UseHttpsRedirection();` ?

Reply:
@DenisKrivoruchko11 could you please explain why you think we should use it?
I googled and it's used when the app is in production. While we're developing, I think it's not necessary.

Reply:
How about  'The structure should be like that: .../Services/ProjectService' ?

Reply:
@ifelseelif @djino97 @DenisKrivoruchko11 @KostinP @lalagvana 
I think we can discuss the structure on the next call at 15.30